UsefulProteomicsDatabases/PtmListLoader.cs

```diff
@@ -138,6 +139,7 @@ private static IEnumerable<Modification> ReadMod(String ptmListLocation, List<st
                         {
                             //This catches a bug where there is a correct two letter code entry but no information that follows. so, when trim get's at it, the string is not at least 5 characters and then there is a crash.
```

Comment:
I still don't see how this isn't caught by the `if (line.Length > 5)`, but I don't have a correction to suggest.